### ISSUE TITLE
Embracing cohort heterogeneity in clinical machine learning development: a step toward generalizable models

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#14]
## TL;DR

Paper discusses the benefit of averaging over cohorts, rather than developing a prediction model from a single cohort. Showing that models trained on data from multiple cohorts can perform significantly better in new settings than models based on the same amount of training data but from just a single cohort. Although this concept seems simple and obvious, no current prediction model development guidelines recommend such an approach.

### Paper Link

https://www.nature.com/articles/s41598-023-35557-y

### Author/Institution

Michiel Schinkel (University of Amsterdam)

## Cite

Schinkel, M., Bennis, F.C., Boerman, A.W. et al. Embracing cohort heterogeneity in clinical machine learning development: a step toward generalizable models. Sci Rep 13, 8363 (2023). https://doi.org/10.1038/s41598-023-35557-y
